### DISCUSSION TITLE
Custom Mouse Pointer Blocks

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/packager#425] [Ideas]
Currently, you can only hide/lock/change the mouse pointer from the packager options, which makes it impossible if you want to change it in the middle of running the project. I suggest new blocks to change this.
![scratchblocks](https://user-images.githubusercontent.com/96546098/173197515-619ee912-3305-4dc6-87a3-4167228b1111.png)

**Comment by TheShovel** [comment]
It is not impossible #394
